src/main/java/org/broadinstitute/hellbender/utils/IntervalMergingRule.java

```diff
@@ -5,5 +5,6 @@
  */
 public enum IntervalMergingRule {
     ALL, // we merge both overlapping intervals and abutting intervals
-    OVERLAPPING_ONLY // We merge intervals that are overlapping, but NOT ones that only abut each other
+    OVERLAPPING_ONLY, // We merge intervals that are overlapping, but NOT ones that only abut each other
+    NONE // Don't perform any interval merging at all. This will likely cause problems for the traversal and this is not recommended
```

Comment:
Can you add a note to the effect that `NONE` is advanced and may not work for all traversal types to the doc string for the actual `--interval-merging-rule` argument in `IntervalArgumentCollection` as well?

Reply:
"This will likely cause problems for the traversal and this is not recommended" -> "This may cause problems with some traversal types and should only be used when a particular tool indicates that it is safe for use with that tool"

Reply:
This has been removed, now this branch only includes a method for getting the intervals unmerged and associated tests.